pkg/collector/corechecks/snmp/internal/checkconfig/config_validate_enrich.go

```diff
@@ -175,6 +175,26 @@ func validateEnrichMetricTag(metricTag *profiledefinition.MetricTagConfig) []str
 	if metricTag.Column.OID != "" || metricTag.Column.Name != "" {
 		errors = append(errors, validateEnrichSymbol(&metricTag.Column, MetricTagSymbol)...)
 	}
+
+	// OID/Name to Symbol harmonization:
+	// When users declare metric tag like:
+	//   metric_tags:
+	//     - OID: 1.2.3
+	//       name: aSymbol
```

Comment:
`symbol: aSymbol`

Reply:
indeed, it's typo, fixed here https://github.com/DataDog/datadog-agent/pull/19925/commits/749922bee7f7ab11777ca7a16aeb066bd5af1e5b